apps/vr-tests-react-components/src/stories/Persona.stories.tsx

```diff
@@ -86,16 +86,26 @@ storiesOf('Persona Converged', module)
   .addStory('textAlignment', () => (
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26546&commitId=28e2ef98a1b63f4519e0bc39907dd60a86314537&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1868 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Persona Converged.textAlignment.normal.chromium.png|15317| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>